README.md

```diff
@@ -7,14 +7,29 @@ Built With
 CSS
 HTML
 
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [Optional] Consider adding steps required for one to get a local copy of your project under this section. This will make your work more professional.